src/eko/mellin.py

```diff
@@ -233,12 +233,15 @@ def __init__(self, t, logx, axis_offset):
     # TODO: make also the other 2 paths available ??
     @property
     def n(self):
+        """Returns the Mellin moment N"""
         return Talbot_path(self.t, self.r, self.o)
 
     @property
     def jac(self):
+        """Returns the Jacobian of the Mellin path"""
         return Talbot_jac(self.t, self.r, self.o)
 
     @property
     def prefactor(self):
+        r"""Returns the mellin inversion prefactor :math:`-\frac{1}{\pi}`"""
```

Comment:
it is `1/(i*pi) = - i/pi` (the text book value beeing `1/(2pi i)`, but we're even folding the integral https://eko.readthedocs.io/en/latest/theory/Mellin.html#mellin-space-and-transformations )

Reply:
yes! I forgot the `i`